src/frontend/screens/MapScreen/index.tsx

```diff
@@ -148,13 +151,15 @@ export const MapScreen = ({
         />
 
         {coords && locationServicesEnabled && (
-          <UserLocation minDisplacement={MIN_DISPLACEMENT} />
+          <UserLocation minDisplacement={TRACKING_DISTANCE_INTERVAL} />
         )}
 
+        {isTracking && <UserTooltipMarker />}
```

Comment:
potentially out of scope: wondering if the rendering of these should be placed after the rendering of the different map layers as in move these lines further down in the file.

maybe something smart happens under the hood but otherwise the typical behavior is that things applied later overlap on top of previously applied UI elements in the map. Generally seems more reasonable for the user location and the tooltip to be displayed on top of everything else.